Demo/Application/Features/Venmo - Custom Button/BraintreeDemoVenmoViewController.swift

```diff
@@ -0,0 +1,49 @@
+import UIKit
+import BraintreeVenmo
+
+class BraintreeDemoVenmoViewController: BraintreeDemoPaymentButtonBaseViewController {
+    
+    var venmoClient: BTVenmoClient!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        venmoClient = BTVenmoClient(apiClient: self.apiClient)
+        title = "Custom Venmo Button"
+    }
+    
+    override func createPaymentButton() -> UIView! {
+        let button = UIButton(type: .custom)
+        button.setTitle("Venmo (custom button)", for: .normal)
+        button.setTitleColor(.systemBlue, for: .normal)
+        button.addTarget(self, action: #selector(tappedCustomVenmo), for: .touchUpInside)
+        return button
+    }
+    
+    @objc func tappedCustomVenmo() {
+        self.progressBlock("Tapped Venmo - initiating Venmo auth")
+        
+        let venmoRequest = BTVenmoRequest(paymentMethodUsage: .multiUse)
+        venmoRequest.vault = true
+        venmoRequest.collectCustomerBillingAddress = true
+        venmoRequest.collectCustomerShippingAddress = true
+        venmoRequest.totalAmount = "30.00"
+        venmoRequest.taxAmount = "1.10"
+        venmoRequest.discountAmount = "1.10"
+        venmoRequest.shippingAmount = "0.00"
```

Comment:
I wonder if we should do something like we have in the NXO ViewController and have a separate button for the standard vs the enriched customer data flow. Would be an easy way to test both flows and could have prevented the dictionary bug if we had both buttons on the same screen!

Reply:
Done!